lib/yfom/value/osm_geocode.php

```diff
@@ -147,6 +161,7 @@ public function getDefinitions(): array
                 ['customfunction' => ['name' => 'latlng', 'function' => $this->validateLatLng(...)]],
                 ['customfunction' => ['name' => 'address', 'function' => $this->validateAddress(...)]],
                 ['customfunction' => ['name' => 'no_db', 'function' => $this->validateNoDb(...)]],
+                ['customfunction' => ['name' => ['height','format'], 'function' => $this->validateLayout(...)]],
```

Comment:
BC: den Parameter eins nach hinten schieben ändert dann nicht die bestehende Reihenfolge..

Reply:
Ist m.E. unkritisch, denn den auch der letzte Parameter ist gestern oder vorgestern ert von mir eingebaut worden und nicht in einem Release veröffentlicht. Den kann noch niemand gentzt haben, daher kein BC.